Plugin.php

```diff
@@ -163,25 +168,25 @@ public function registerNotificationRules()
                 ],
             ],
             'events' => [
-                \RainLab\User\NotifyRules\UserActivatedEvent::class,
-                \RainLab\User\NotifyRules\UserRegisteredEvent::class,
+               \Winter\User\NotifyRules\UserActivatedEvent::class,
+               \Winter\User\NotifyRules\UserRegisteredEvent::class,
             ],
             'actions' => [],
             'conditions' => [
-                \RainLab\User\NotifyRules\UserAttributeCondition::class
+                'Winter\User\NotifyRules\UserAttributeCondition',
```

Comment:
Why was this switched to not use the class constant?

Reply:
Fixed.